actions/icinga.schedule.downtime.yaml

```diff
@@ -0,0 +1,45 @@
+---
+description: Schedule a downtime for Host or Service in Icinga
+enabled: true
+entry_point: src/openstack_actions.py
+name: icinga.schedule.downtime
+parameters:
+  name:
+    type: string
+    description: Name of host or service in Icinga
+    required: true
+  start_time:
+    type: string
+    description: "Date and time (UTC) in the format: %d/%m/%y %H:%M:%S"
+    required: true
+  duration:
+    type: integer
+    description: Duration of downtime in seconds
+    required: true
```

Comment:
We can specify the start time using DMY HMS, but then if I want to have a 1 week downtime I have to calculate days -> seconds
Any reason the end time can't also use the date too?

Reply:
Added end_time patameter